schema_input.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/rnaseq/master/schema_input.json",
```

Comment:
Don't we need to version control these URLs with the pipeline release tag? e.g. to run older pipeline versions if required.

Reply:
Yes we probably should do, I guess we can do that with `bump-versions`. In practice I don't think that the `$id` attribute is actually used by anything anywhere. We could even just remove it, although that would be against best-practices for JSON Schema.

https://json-schema.org/understanding-json-schema/structuring.html#id